manifests/g/GIGABYTE/GigabyteControlCenter/25.02.07.01/GIGABYTE.GigabyteControlCenter.installer.yaml

```
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.4.3 $debug=QUSU.CRLF.7-5-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GIGABYTE.GigabyteControlCenter
+PackageVersion: 25.02.07.01
+InstallerType: zip
+InstallModes:
+  - interactive
+  - silent
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /silent
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+  - DisplayName: GIGABYTE Control Center 25.01.20.01
+    Publisher: GIGABYTE
+    DisplayVersion: 25.02.07.01
```

```suggestion
    DisplayVersion: 25.01.20.01
```

您能否再检查一下此条目？我(在没有测试的情况下)认为是这个

---

✅ 已确认，之前截图里显示的版本号也是这个。

😕 非常不理解为什么安装后显示的版本号和 `InstallerUrl` 中标注的版本号不一样。

- 参见 <https://github.com/microsoft/winget-pkgs/pull/230284#issue-2856223464>